src/codeu/chat/client/simplegui/SignInPanel.java

```diff
@@ -0,0 +1,59 @@
+package codeu.chat.client.simplegui;
+
+/**
+ * Created by Suveena on 5/18/17.
+ */
+
+import javax.swing.*;
+import java.awt.*;
+
+/*import java.awt.event.ActionEvent;
```

Comment:
ditto commented out code

If you use eclipse, there's a feature that will help you organize all your imports and perform auto-imports so it's pretty easy to get them back

Also if you want to keep different versions and play around, easy to make a new branch in git